### ISSUE TITLE
False positive: "Component Routing is not configured to load its targets asynchronously" in sap.fe application

### ISSUE THREAD

**Issue by tobiasqueck** [SAP/ui5-linter#184]
When running the linter in a newly generated Fiori elements application then an error is shown in the `Component.js`. The error indicates that the component needs to implement the interface `sap.ui.core.IAsyncContentCreation`, however, the interface is already implemented by the base class (`sap/fe/core/AppComponent`), therefore, it shouldn't be necessary.

## Expected Behavior

No error in the `Component.js`

## Current Behavior

```
6:16 **error** Component Routing is not configured to load its targets asynchronously.. Details: Use Asynchronous Loading (https://ui5.sap.com/#/topic/676b636446c94eada183b1218a824717). Implement sap.ui.core.IAsyncContentCreation interface in Component.js or set the "async" flag for "sap.ui5/routing/config" in the component manifest.
```

## Steps to Reproduce the Issue
1. Use the Fiori application generate and generate a LROP application for an OData v4 service
2. Run `npx --yes @ui5/linter@latest --details`
3. Check the console

## Context

- UI5 linter version: **`latest`**
- Node.js Version: **`unknown`**
- npm Version: `unknown`
- OS/Platform: `unknown`
- Browser *(if relevant)*: `unknown`
- Other information regarding your environment *(optional)*: `nothing`

## Log Output / Stack Trace

```
{...}
```

**Comment by tobiasqueck** [comment]
@nlunets @petermuessig here is the continuation of our internal discussion.

**Comment by tobiasqueck** [comment]
@RandomByte could you please let me know which version of the linter contains the fix so that I can retest. I'd also appreciate if you could keep the issue open until the fix is verified.

**Comment by RandomByte** [comment]
The proposed fix is released with [v0.2.6](https://github.com/SAP/ui5-linter/releases/tag/v0.2.6)

**Comment by tobiasqueck** [comment]
Perfect, I can confirm that it works as expected
```
~/tmp/project2 (main)> npx --yes @ui5/linter@0.2.5 --details
UI5 linter report:

~/tmp/project2/webapp/Component.js
  6:16 error Component Routing is not configured to load its targets asynchronously.. Details: Use Asynchronous Loading (https://ui5.sap.com/#/topic/676b636446c94eada183b1218a824717). Implement sap.ui.core.IAsyncContentCreation interface in Component.js or set the "async" flag for "sap.ui5/routing/config" in the component manifest.

1 problems (1 errors, 0 warnings)
~/tmp/project2 (main)> npx --yes @ui5/linter@0.2.6 --details 
UI5 linter report:

0 problems (0 errors, 0 warnings)
```

**Comment by RandomByte** [comment]
Thanks for reporting and confirming the fix!
